deploy/README.md

```diff
@@ -0,0 +1,309 @@
+
+#### docker-compose.yml
+To further customize your instance, consider adjusting `docker-compose.yml`. In order to ease updating to newer versions of the WBS Deployment Kit, consider putting your customization into a new file called `docker-compose.override.yml` and from now on, restart your instance with
+
+```sh
+docker compose -f docker-compose.yml -f docker-compose.override.yml down
+docker compose -f docker-compose.yml -f docker-compose.override.yml up --wait
+```
+
+This way your changes are kept separate from the original WBS Deploying Kit code.
+
+### Managing your Data
+Besides [your configuration](#configuring-your-wikibase-suite), your data is obviously what makes your instance unique. All data is stored in [Docker Volumes](https://docs.docker.com/storage/volumes/).
+
+ - `wikibase-image-data` MediaWiki image and media file uploads.
+ - `mysql-data` MediaWiki/Wikibase MariaDB raw database.
+ - `wdqs-data` Wikidata Query Service raw database.
+ - `elasticsearch-data` Elastic Search raw database.
+ - `quickstatements-data` Generated Quickstatement OAuth binding for this MediaWiki instance.
+ - `traefik-letsencrypt-data` SSL certificates.
+
+#### Backup your Data
+You can backup your data by shutting down the instance and dumping the contents of all Docker Volumes into `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker run --rm --volume $v:/backup debian:12-slim tar cz backup > $v.tar.gz
+done
+```
+
+#### Restore from a backup
+
+To restore the volume backups, ensure your instance is shut down with `docker compose down` and populate the Docker Volumes with data from your `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker volume rm $v 2> /dev/null
+  docker volume create $v
+  docker run -i --rm --volume $v:/backup debian:12-slim tar xz < $v.tar.gz
+done
+```
+
+### Updating
+
+WBS is versioned with [Semantic Versioning](https://semver.org/spec/v2.0.0.html). The WBS Deployment Kit and all the WBS Service Images have individual version numbers.
+
+> 💡 WBS Deployment Kit always references the latest minor and patch releases of the compatible WBS Service Images major versions.
+
+#### Minor and patch updates for WBS Service Images
+
+As WBS Deployment Kit always references latest minor and patch releases of compatible WBS Service Images TODO: rephrase, non breaking changes (including security updates) are applied automatically when recreating Docker containers. This should be always safe to do. Simply run
```

Comment:
TODO...

Reply:
leaving it here as a reminder to myself

Reply:
Thanks fixed.

Reply:
@adeeri I now put a whole paragraph with an example how the updating works. I think this is what you suggested earlier. Let me know if this works for you https://github.com/wmde/wikibase-release-pipeline/tree/T362226-example-docs-2/deploy#updating-and-versioning

Reply:
marking this as resolved for now. feel free to provide feedback and suggest/make changes at any time @adeeri